sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/src/autorest.md

```diff
@@ -0,0 +1,17 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/f09aacf4c6b63be416212cb182f6b31e8bc6d545/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/preview/2023-06-01-preview/hdinsight.json
```

Comment:
Could we change this to use the readme file as input instead?
https://github.com/Azure/azure-rest-api-specs/blob/f09aacf4c6b63be416212cb182f6b31e8bc6d545/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md
This will enable the automatically update in `azure-sdk-for-net` in the swagger repo if we have a new PR there and preview your changes in dotnet SDK.

Reply:
Hi @ArcturusZhang , we can not use the readme.md under that commit because the readme.md file need to be updated otherwise the library name or the management client name is not correct. 
The https://github.com/Azure/azure-rest-api-specs/blob/f09aacf4c6b63be416212cb182f6b31e8bc6d545/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md is not the same wit the approved namespace.

I have new PR to update the readme file, it hasn't been merged please see https://github.com/Azure/azure-rest-api-specs/pull/25321

Reply:
I think I can update it once my update readme file PR is merged.

Reply:
Good news is that the update readme file is merged today. I though it will take several days. Updated.